rllab/misc/logger.py

```diff
@@ -16,6 +16,8 @@
 import pickle
```

Comment:
can you move these commits into a different PR? On github that requires that you use a separate branch.

note that the CI is failing because it requires that you reformat any file you touch to be PEP8 format.